src/MapBox.js

```diff
@@ -42,85 +47,152 @@ function MapBox(props) {
     }
 
     const initMap = ({setMap, mapContainer}) =>{
-      const mMap = new mapboxgl.Map({
+      
+      let mapOption = {
         container: mapContainer.current,
-        style: 'mapbox://styles/mapbox/streets-v11',
+        style: (props.map.canvas.style!==undefined)? mapboxStyle[props.map.canvas.style]: 'mapbox://styles/mapbox/streets-v11',
         center: props.map.camera.center,
-        zoom: props.map.camera.zoom
-      });
+        zoom: props.map.camera.zoom,
+      }
+      if(props.map.camera.pitch!==undefined)
+        mapOption.pitch = props.map.camera.pitch;
+      if(props.map.camera.bearing!==undefined)
+        mapOption.bearing = props.map.camera.bearing;
+      if(props.slideshow!==undefined)
+      {
+        mapOption.maxZoom = props.slideshow.maxZoom
+        mapOption.minZoom = props.slideshow.minZoom 
+      }
+      
+      const mMap = new mapboxgl.Map(mapOption);
+      
       
+      if(!props.map.canvas.scrollZoom)
+        mMap.scrollZoom.disable();
+
       mMap.on('load', () =>{
         mMap.resize();
         setMap(mMap);
         
       });
+      
     };
 
     const drawMarkers = ()=>{
```

Comment:
Please add a space between () and =>, => and { for readability.

Reply:
Okay, I added spaces in all of arrow function expression and comparison operators

jeesubKim/stickboard :  4cf7530c5f316f45dc32d28747e69729b94d3a8c